components/style/properties/gecko.mako.rs

```diff
@@ -2235,6 +2235,36 @@ fn static_assert() {
     }
 
     <% impl_shape_source("shape_outside", "mShapeOutside") %>
+
+    pub fn set_contain(&mut self, v: longhands::contain::computed_value::T) {
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_NONE;
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_STRICT;
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_LAYOUT;
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_STYLE;
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_PAINT;
+        use gecko_bindings::structs::NS_STYLE_CONTAIN_ALL_BITS;
+        use properties::longhands::contain;
+
+        self.gecko.mContain = if v.contains(contain::STRICT) {
+            NS_STYLE_CONTAIN_STRICT | NS_STYLE_CONTAIN_ALL_BITS
```

Comment:
On top of the bit @heycam spotted, I see that gecko splits `STRICT` and `ALL_BITS`, presumably for serialization purposes (so `contain: layout style paint` doesn't serialize as `contain: strict`). Can you verify it?

If so, we should ensure the serialization behavior is the same, and add an `ALL_PROPERTIES` bit to Servo too. Once that's done, we can test that the constants are the expected ones with unit tests (see the `assert_restyle_hints_match` test, and this can just become:

```rust
self.gecko.mContain = v.bits;
```

Does that sounds reasonable?